app/controllers/reputation_controller.rb

```diff
@@ -29,7 +37,11 @@ def advertisement
     if user_signed_in?
       @advertisement = Advertisement.find(params[:id])
       @evaluation_value = params[:up_down] == 'down' ? 0 : 1
-      @advertisement.add_or_update_evaluation(:likes, @evaluation_value, current_user)
+      if params[:up_down] == 'down'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.